sdk/core/Azure.Core.TestFramework/src/RecordedTestBase.cs

```diff
@@ -134,6 +134,24 @@ public void StopLoggingEvents()
             Logger = null;
         }
 
+        [OneTimeSetUp]
+        public void UpdateDefaultPollingTime()
+        {
+            if (Mode == RecordedTestMode.Playback)
+            {
+                foreach (var assembly in AppDomain.CurrentDomain.GetAssemblies())
```

Comment:
We wouldn't need the assembly lookup if the `OperationHelpers` class was declared as part of the Core library (instead of being manually added as a shared resource). Is there any particular reason for making it the way it is?

Reply:
It wasn't an API that we wanted to publicly expose to customers.

Reply:
But it wouldn't be a problem since `OperationHelpers` is internal, right?

Reply:
How would you use it from other libraries if it stays internal?

Reply:
Good point.